packages/lexical-playground/src/plugins/CommentPlugin/index.tsx

```diff
@@ -930,6 +904,43 @@ export default function CommentPlugin({
           }
         });
       }),
+      // style text nodes with comment IDs
+      editor.registerUpdateListener((payload) => {
+        const {mutatedNodes} = payload;
+        editor.getEditorState().read(
```

Comment:
Nit: if you're trying to access stuff from the DOM, there are some util functions that fail if there's not an active editor. You need to use `editor.read()` or `editor.update()` rather than reading from a static `EditorState`.

```suggestion
        editor.read(
```